benchmark/src/client.rs

```diff
@@ -143,6 +143,17 @@ impl<B: Backoff> ExecutorContext<B> {
     }
 }
 
+// Since impl trait is not stable yet, implement this as a function is impossible without box.
+macro_rules! spawn {
```

Comment:
What's the difference between the `fn spawn()` above?

Reply:
`client` can be different types.

Reply:
Could we remove the above `fn spawn()` ?